### PR TITLE
Remove para module paths for ncdiag on WCOSS2

### LINKED ISSUE #1426: Remove para module paths for ncdiag on WCOSS2
**Description**

From the March 24th RFC memo:
```
RFC 10769 - On WCOSS2, install the ncio v1.1.2 and ncdiag v1.0.0 libraries into
production. The ncio lib (GSI IO Library) and ncdiag lib (GSI NetCDF
Diagnostics Library) will be used for upcoming model upgrades such as HAFS
(Hurricane Analysis and Forecast System). To be implemented on March 27 at
1500Z.
```

With `ncdiag/1.0.0` moving from para to prod on WCOSS2 we can now update the `ncdiag` lines in `module_base.wcoss2.lua` to remove the para paths: https://github.com/NOAA-EMC/global-workflow/blob/develop/modulefiles/module_base.wcoss2.lua#L33:L35

**Acceptance Criteria (Definition of Done)**

Modulefile is updated and the `ncdiag` module is accessible via its production location.

**Dependencies**

- [x] RFC 10769 is completed.
- [x] GSI update: https://github.com/NOAA-EMC/GSI/issues/554 & https://github.com/NOAA-EMC/GSI/pull/555
- [x] GSI-monitor update: https://github.com/NOAA-EMC/GSI-Monitor/issues/74 & https://github.com/NOAA-EMC/GSI-Monitor/pull/75

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
**Description**

This PR updates the `ncdiag` module for WCOSS2 from the para version to the newly implemented prod version. The `module_base.wcoss2.lua` modulefile is updated. New hashes for GSI (`113e307`) and GSI-Monitor (`8cf16de`) for the same update are included. Also moved the `crtm/2.4.0` module down to be consistent with the module load order we have in our other modulefiles.

Resolves #1426

**Type of change**

Operational update maintenance.

**How Has This Been Tested?**

- [x] Clone and build tests on WCOSS2-Cactus
- [x] Cycled C96C48L127 test on WCOSS2
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] Any dependent changes have been merged and published

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
looks good.

File: modulefiles/module_base.wcoss2.lua [hunk 1]
```diff
@@ -26,16 +26,11 @@ load(pathJoin("prod_util", "2.0.9"))
...
+load(pathJoin("crtm", "2.4.0"))
 load(pathJoin("wgrib2", "2.0.7"))
 
-pushenv("HPC_OPT", "/apps/ops/para/libs")
-append_path("MODULEPATH", "/apps/ops/para/libs/modulefiles/compiler/intel/19.1.3.304")
-append_path("MODULEPATH", "/apps/ops/para/libs/modulefiles/mpi/intel/19.1.3.304/cray-mpich/8.1.7")
-
-load("ncdiag/1.0.0")
-
 prepend_path("MODULEPATH", pathJoin("/lfs/h2/emc/global/save/emc.global/git/prepobs/v1.0.1/modulefiles"))
```
Comment (RussTreadon-NOAA) [inline modulefiles/module_base.wcoss2.lua:34]: Not part of this PR but why do we pick up `prepobs/1.0.1` from a non-production space?

File: modulefiles/module_base.wcoss2.lua [hunk 2]
```diff
@@ -26,16 +26,11 @@ load(pathJoin("prod_util", "2.0.9"))
 load(pathJoin("grib_util", "1.2.3"))
 load(pathJoin("bufr_dump", "1.0.0"))
 load(pathJoin("util_shared", "1.4.0"))
-load(pathJoin("crtm", "2.4.0"))
 load(pathJoin("g2tmpl", "1.9.1"))
+load(pathJoin("ncdiag", "1.0.0"))
+load(pathJoin("crtm", "2.4.0"))
```
Comment (RussTreadon-NOAA) [inline modulefiles/module_base.wcoss2.lua:31]: Just curious.  Why was the crtm line moved?

**Review by RussTreadon-NOAA — COMMENTED** [review]
Changes look fine.   Two questions out of curiosity.

File: modulefiles/module_base.wcoss2.lua (hunk 2 above)
Comment (KateFriedman-NOAA) [reply modulefiles/module_base.wcoss2.lua:31]: I moved the `crtm/2.4.0` to be consistent with the module load order we have in our other modulefiles. I failed to mention this in the PR description. I'll correct that, thanks!

Comment (RussTreadon-NOAA) [reply modulefiles/module_base.wcoss2.lua:31]: Consistency ... I like this!

File: modulefiles/module_base.wcoss2.lua (hunk 1 above)
Comment (KateFriedman-NOAA) [reply modulefiles/module_base.wcoss2.lua:34]: We could certainly set this to use the officially installed production `prepobs/1.0.1` module on WCOSS2. Using a non-prod/development install provides a tad more flexibility though for development testing. Since this modulefile (like the others) is used for non-production/developer runs, I have us pointing to our own install. This is also consistent with what we do on the other platforms and is a historical thing (we used to point to non-prod installs of prepobs in `config.base` and still do for obsproc). When prepobs became a module we moved it from `config.base` to the modulefiles.

When we install newer prepobs versions in our own space, that may not be in production, we can update the version on this line. Once version files come into play in `develop` we'll just change the prepobs version in the version file and leave this line as is with a version variable. We will have a development version of prepobs in the near future to support the GFS post-com-reorg and won't be able to use the production module so we'll need the development install for a while.

Thinking ahead, when we plan a tag for ops that uses the production prepobs we can change this to use that (assuming it's already available and installed in production space). But this modulefile is not used in production anyway. By then we should have also broken the module_base files apart and set module loads per job, so this prepobs module load will have moved to wherever our development prep job sources its modules from.

Hope the above explains some of the history and reasoning in not picking up `prepobs` module from production space when on WCOSS2. Good question. :)

Comment (RussTreadon-NOAA) [reply modulefiles/module_base.wcoss2.lua:34]: Thank you for the detailed explanation.   It will be nice when module loads are at the job level.   This is consistent with operations.

**Review by RussTreadon-NOAA — APPROVED** [review]
Approve.

File: modulefiles/module_base.wcoss2.lua (hunk 1 above)
Comment (KateFriedman-NOAA) [reply modulefiles/module_base.wcoss2.lua:34]: > It will be nice when module loads are at the job level. This is consistent with operations.

Agreed!

**Review by WalterKolczynski-NOAA — APPROVED** [review]
